.packer/template.pkr.hcl

```diff
@@ -0,0 +1,38 @@
+packer {
+  required_plugins {
+    amazon = {
+      version = ">= 0.0.2"
+      source  = "github.com/hashicorp/amazon"
+    }
+  }
+}
+
+source "amazon-ebs" "source" {
+  ami_name      = "Deep Learning AMI Neuron PyTorch - {{timestamp}}"
+  ami_users     = [
+    "844815912454",
+    "382373577178",
+    "451934005581"
+  ]
+  instance_type = "trn1.2xlarge"
+  ssh_username = "ec2-user"
+  region        = "us-east-1"
+  source_ami_filter {
+    filters = {
+      name   = "Deep Learning AMI Neuron PyTorch 1.13*" # Pin to a specific version for now
```

Comment:
Potentially we can move away from this base image. I believe the most interesting part of it is the installed kernel modules to enable amazon linux to use gpus.